reana-ui/src/actions.js

```diff
@@ -293,3 +308,24 @@ export function fetchWorkflowSpecification(id) {
     return resp;
   };
 }
+
+export function deleteWorkflow(id, workspace = false) {
+  return async (dispatch) => {
+    dispatch({ type: WORKFLOW_DELETE_INIT });
+    return await axios
+      .put(
```

Comment:
Just wondering which will be best here, `PUT` or `PATCH` since we are just changing the status of workflow and not actually deleting it. Please correct me if I am wrong.

Reply:
I think you're right. If it were complete deletion we should use `DELETE` though. In this case, as we're only sending the new `status` and not the full workflow object it should be a [`PATCH`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH).

I left the API untouched as it was, I don't know if it's worth changing it at the moment as it'd affect other places. Opinions? @tiborsimko @diegodelemos @audrium

Reply:
+1 for changing it, but maybe we could create another task to change it sometime in the future as it's quite minor thing IMO

Reply:
> +1 for changing it, but maybe we could create another task to change it sometime in the future as it's quite minor thing IMO

+1

This is a good example to use our [Triage board](https://github.com/orgs/reanahub/projects/7). Creating an issue stemming from this comment, tagging it as refactor, and adding it to `Dev next/future` depending on the urgency. @ParthS007 do you want to take care of it? Not to forget to look for existing issues that might be related such as https://github.com/reanahub/reana-workflow-controller/issues/143.

Reply:
Yes, I can have a look at it. Not sure how many other places will be affected by this.